util/customTransformer-v1.js

```diff
@@ -0,0 +1,96 @@
+const ivm = require("isolated-vm");
+const fetch = require("node-fetch");
+const fs = require("fs");
+const lodashCore = require("lodash/core");
+const _ = require("lodash");
+const stats = require("./stats");
+
+// TODO: Check why these dont work
+const unsupportedFuncNames = [
+  "_",
+  "extend",
+  "each",
+  "first",
+  "join",
+  "reverse",
+  "split"
+];
+
+const { getPool } = require("./ivmPool");
+
+async function transform(isolatevm, events) {
+  const transformationPayload = {};
+  transformationPayload.events = events;
+  transformationPayload.transformationType = isolatevm.fName;
+  const executionPromise = new Promise(async (resolve, reject) => {
+    const sharedTransformationPayload = new ivm.ExternalCopy(
+      transformationPayload
+    ).copyInto({
+      transferIn: true
+    });
+    try {
+      await isolatevm.bootstrapScriptResult.apply(undefined, [
+        isolatevm.fnRef,
+        new ivm.Reference(resolve),
+        sharedTransformationPayload
+      ]);
+    } catch (error) {
+      reject(error.message);
+    }
+  });
+  const timeoutPromise = new Promise(resolve => {
```

Comment:
Make sure we tested this case using an infinite loop transformation function. It should exit in 4 seconds with timeout error